spring-vault-core/src/main/java/org/springframework/vault/support/Certificate.java

```diff
@@ -63,13 +70,14 @@ public class Certificate {
 	 * @param issuingCaCertificate must not be empty or {@literal null}.
 	 * @return the {@link Certificate}
 	 */
-	public static Certificate of(String serialNumber, String certificate, String issuingCaCertificate) {
+	public static Certificate of(String serialNumber, String certificate, String issuingCaCertificate,
```

Comment:
Instead of updating existing factory methods, let's introduce new ones, please. Otherwise, we introduce a breaking change. We generally look for a smooth upgrade experience across minor versions.